src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/services/CaseUpdater.java

```diff
@@ -44,8 +45,25 @@ public SuccessfulUpdateResponse update(CaseUpdateRequest caseUpdateRequest) {
             "Missing scanned documents in exception record"
         );
 
+        if (caseUpdateRequest.isAutomatedProcess) {
+            Assert.notNull(
+                caseUpdateRequest.caseUpdateDetails,
+                "Missing case update details when process is automated"
```

Comment:
message sounds not good to me.
`Case update details is required for automated  process`